project-base/src/Shopsys/ShopBundle/Migrations/Version20191121171000.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20191121171000 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->sql('DELETE FROM friendly_urls WHERE slug=\'contact\'');
+        $this->sql('DELETE FROM articles WHERE name=\'Contact\' AND placement=\'footer\'');
```

Comment:
Maybe separating migration to two can help if someone wants to delete contact slug but keep the article, what do you think?

Reply:
I didn't split it as I asked them to move old stuff to the new contact page